src/components/IssueEdit.vue

```diff
@@ -253,7 +255,7 @@ function addPublicComment() {
             <LabelSelect label="Source" :options="flawSources" v-model="flawSource" :error="errors.source"/>
           </div>
           <div class="col-6">
-            <div class="text-end">Major Incident: {{ flaw.is_major_incident ? 'YES' : 'No' }}</div>
+            <LabelSelect label="Major Incident" :options="incidentTypes" v-model="flawMajor_incident_state" :error="errors.major_incident_state"/>
```

Comment:
"Major Incident" is not an accurate label anymore

Reply:
Changed label and vars to "Incident State"